contracts/test/mocks/GatewayMock.sol

```diff
@@ -63,6 +63,10 @@ contract GatewayMock is Gateway {
             return commitmentsAreVerified;
         }
     }
+
+    function setTokenTransferFeesPublic(bytes calldata params) external {
```

Comment:
What does the `Public` mean here?

Reply:
It's a wrapper function/pattern we use to test the `onlySelf` executable in gateway contract.